app/src/main/res/layout/fragment_complaint_status.xml

```diff
@@ -0,0 +1,305 @@
+                        style="@style/OpenSansSemiBold20sp"
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        android:layout_marginHorizontal="5dp"
+                        android:layout_marginTop="10dp"
+                        tools:text="@string/category"
+                        android:textColor="@color/black"
+                        app:layout_constraintEnd_toStartOf="@+id/cardTickedID"
+                        app:layout_constraintStart_toStartOf="parent"
+                        app:layout_constraintTop_toTopOf="parent" />
+
+                    <TextView
+                        android:id="@+id/localityTV"
+                        style="@style/OpenSansRegular16sp"
+                        android:layout_width="0dp"
+                        android:layout_height="wrap_content"
+                        android:layout_marginTop="5dp"
+                        tools:text="@string/locality"
+                        android:textColor="@color/grey_4F4F4F"
+                        app:layout_constraintEnd_toEndOf="@+id/categoryTV"
+                        app:layout_constraintHorizontal_bias="0.0"
+                        app:layout_constraintStart_toStartOf="@+id/categoryTV"
+                        app:layout_constraintTop_toBottomOf="@+id/categoryTV" />
+
+                    <androidx.cardview.widget.CardView
+                        android:id="@+id/cardTickedID"
+                        android:layout_width="45dp"
+                        android:layout_height="35dp"
+                        android:layout_marginTop="10dp"
+                        android:layout_marginEnd="5dp"
+                        app:cardBackgroundColor="@color/card_color"
+                        app:cardCornerRadius="10dp"
+                        app:cardElevation="5dp"
+                        app:layout_constraintEnd_toEndOf="parent"
+                        app:layout_constraintTop_toTopOf="parent">
+
+                        <TextView
+                            android:id="@+id/complaintIdTV"
+                            style="@style/OpenSansSemiBold12sp"
+                            android:layout_width="wrap_content"
+                            android:layout_height="wrap_content"
+                            android:layout_gravity="center"
+                            android:clickable="false"
+                            android:enabled="false"
+                            tools:text="@string/ticketID_example"
+                            android:textColor="@color/black" />
+
+                    </androidx.cardview.widget.CardView>
+
+                    <ImageView
+                        android:id="@+id/imageView"
+                        android:layout_width="100dp"
+                        android:layout_height="100dp"
+                        android:layout_marginTop="12dp"
+                        app:layout_constraintEnd_toEndOf="@+id/localityTV"
```

Comment:
you can remove layout_constraintHorizontal_bias

Reply:
Resolved.